core/runtime/binaryen/runtime_external_interface.hpp

```diff
@@ -52,7 +54,13 @@ namespace kagome::runtime::binaryen {
                         size_t expected,
                         size_t actual);
 
+    void methodsRegistration();
+
     std::shared_ptr<host_api::HostApi> host_api_;
+    boost::unordered_map<std::string,
+             std::function<wasm::Literal(wasm::Function *import,
+                                         wasm::LiteralList &arguments)>>
```

Comment:
May store raw pointer to instantiated template function instead of std::function.  
```suggestion
             wasm::Literal(*)(RuntimeExternalInterface &this_, wasm::Function *import,
                                         wasm::LiteralList &arguments)>
```